doc/nrf/links.txt

```diff
@@ -1053,6 +1053,7 @@
 .. _`CSA Certified Products Database`: https://csa-iot.org/csa-iot_products/
 .. _`CSA blog post`: https://csa-iot.org/newsroom/matter-1-1-release-enhancements-for-developers-and-devices/
 .. _`Distributed Compliance Ledger`: https://webui.dcl.csa-iot.org/
+.. _`CSA Matter Resource Kit`: https://groups.csa-iot.org/wg/all-users/home/matter-resource-kit
```

Comment:
Link already exists on Line 1043. You can add this as another a second name for the link, similar to how Zigbee Alliance and Connectivity Standard Alliance have the same target on Lines 1038-1039.

Reply:
Oh, I haven't noticed it's already there, I updated it following the suggestion.